src/service/common/rmm.js

```diff
@@ -22,6 +23,31 @@ exports.recordInteraction = (agent, interaction) => {
 	notifications.notifyInteractions(agent);
 }
 
+exports.addCampaign = (agent, campaign) => {
+	if(!campaignsByAgent[agent]) {
+		campaignsByAgent[agent] = [];
+	}
+	campaignToUpdate = campaignsByAgent[agent].find(c => { return c.name === campaign.GSW_CAMPAIGN_NAME; });
```

Comment:
You can use shorter code:
.find(c => c.name === campaign.GSW_CAMPAIGN_NAME

Reply:
Fixed